opendbc/car/toyota/fingerprints.py

```diff
@@ -1564,6 +1564,23 @@
       b'8646F2402200\x00\x00\x00\x00',
     ],
   },
+  CAR.LEXUS_RC_TSS2: {
+    (Ecu.engine, 0x700, None): [
+      b'\x018966324C8000\x00\x00\x00\x00',
+    ],
+    (Ecu.abs, 0x7b0, None): [
+      b'\x01F15262426000\x00\x00\x00\x00',
+    ],
+    (Ecu.eps, 0x7a1, None): [
+      b'8965B24481\x00\x00\x00\x00\x00\x00',
+    ],
+    (Ecu.fwdRadar, 0x750, 0xf): [
+      b'\x018821F6201400\x00\x00\x00\x00',
+    ],
+    (Ecu.fwdCamera, 0x750, 0x6d): [
+      b'\x028646F2403100\x00\x00\x00\x00!!!!!!!!!!!!!!!!',
```

Comment:
I think a fair amount of new FW has this !!! suffix, which I blocked adding with our FP bot. If we can reasonably prove it's not malformed and intended, we can continue adding it

Reply:
Any routes from this user have it filled out? Maybe it's inconsistent?

Reply:
Marco's dongle ID is 7f8f479cfa6f392a which you can poke around I guess. I'm going to look at Marco's other routes too and it's possible this could be a byproduct of old code on a fork. We'll see how it FPs again on a up to date and close-to-comma branch.

Reply:
https://useradmin.comma.ai/logs?url=https%3A%2F%2Fcommadata2.blob.core.windows.net%2Fqlog%2F7f8f479cfa6f392a%2F00000003--f2985e67e9%2F0%2Fqlog.bz2%3Fse%3D2025-04-09T01%253A26%253A00Z%26sp%3Dr%26sv%3D2024-08-04%26sr%3Db%26rscd%3Dattachment%253B%2520filename%253D7f8f479cfa6f392a_00000003--f2985e67e9--0--qlog.bz2%26sig%3D5bqjFuiaEjnAjD5I3fFS4ba149FaNZQu5Q%252BP4B9wbDA%253D&event=carParams

Checked one and it has `!`s and then this one doesn't have `!`s.

Reply:
Marco's recent stuff doesn't seem to have `!`s. Just the single preserved route on April 2 which I based the fingerprint on. I think it was probably just a fluke so I'll change the PR to the non-`!` one.